clio-util/src/main/scala/org/broadinstitute/clio/util/model/Location.scala

```diff
@@ -10,11 +10,9 @@ sealed trait Location extends EnumEntry
 object Location extends Enum[Location] {
   override val values: IndexedSeq[Location] = findValues
 
-  case object Unknown extends Location
   case object GCP extends Location
   case object OnPrem extends Location
 
-  val pathMatcher = Map(GCP.toString -> GCP, OnPrem.toString -> OnPrem)
+  val pathMatcher: Map[String, Location] = Location.namesToValuesMap
```

Comment:
Not exactly what I meant... we can delete `pathMatcher` entirely.

Reply:
Yeah, I didn't know about Enum.namesToValuesMap.

Reply:
@danxmoran I actually like this better because the calling code asks for a `PathMatcher` and its not an easy logical train of thought to know that it really means `Map[String, Location]`

Reply:
We probably don't need Location.key either.